agents/sbd/fence_sbd.py

```diff
@@ -394,13 +394,14 @@ def main():
         if DEVICE_INIT != return_code:
             exit(return_code)
 
-    # we check against the defined timeouts. If the pacemaker timeout is smaller
-    # then that defined within sbd we should report this.
-    power_timeout = int(options["--power-timeout"])
-    sbd_msg_timeout = get_msg_timeout(options)
-    if power_timeout <= sbd_msg_timeout:
-        logging.warn("power timeout needs to be \
-                greater then sbd message timeout")
+    if options.get("--disable-timeout", "0").lower() in ["0", "no", "off", "false"]:
+        # we check against the defined timeouts. If the pacemaker timeout is smaller
```

Comment:
Is there a reason why we can't simply check power_timeout > 0 before issuing a warning?
This way it would suppress the warning as well if power_timeout got disabled directly and not via dissable-timeout.

Reply:
Hi Klaus,

I think it's okay to just check that power_timeout is greater than 0.

Do you think that is better?
Checking will be simpler.

Best Regards,
Hideo Yamauchi.

Reply:
+1 to check that it's greater than 0.

Just update old line 401 to `if 0  < power_timeout <= sbd_msg_timeout:`.

Reply:
Hi Oyvind,

Thanks for your comment.

> +1 to check that it's greater than 0.
> 
> Just update old line 401 to `if 0 < power_timeout <= sbd_msg_timeout:`.

Okay!

Thanks,
Hideo Yamauchi.